src/docs/product/accounts/membership.mdx

```diff
@@ -60,36 +61,69 @@ Member roles dictate access within an organization.
+| Can change Project settings                                                                             | X (_Only for projects they have access to_)        |                      |
+| Can add and remove Projects                                                                             | X (_Only for teams they’re an existing Admin for_) |                      |
+| Can edit Global Integrations                                                                            |                                                    |                      |
+| Can add, remove, and change members                                                                     |                                                    |                      |
+| Can invite new members to join (_Invite will land in pending state for managers and owners to approve_) | X                                                  | X                    |
+| Can add and remove teams                                                                                |                                                    |                      |
+| Can add and remove repositories                                                                         | X (_Only for teams they’re an existing Admin for_) |                      |
+| Can change organization settings                                                                        |                                                    |                      |
+| Can remove an organization                                                                              |                                                    |                      |
+| Can assign Team Admin Role                                                                              | X (_Only for teams they’re an existing Admin for_) |                      |
+| Can add, edit, or remove Team Contributors                                                              | X                                                  |                      |
+| Can add Projects for teams (depending on the rules set by the org owner or manager)                     | X                                                  | X                    |
+| Can remove Projects from teams                                                                          | X                                                  |                      |
+
+### Team Admin
+
+- Admin access to teams of which they're members.
+- Can create new teams and Projects, as well as remove Projects and teams of which they're members.
+
+### Team Contributor
+
+- Can view and act on issues.
+- Can join and leave teams (depending on the rules set by the org owner/manager).
+- Can invite new members to join (pending approval from org owner/manager).
+- Can add Projects for team (depending on the rules set by the org owner/manager).
+
```

Comment:
We need to explain edge-cases.

If a project is assigned to two teams (Project Fruits to Team Apple and Team Banana), what happens if I'm team-admin for Team Apple, and team-contributor for Team-Banana?